src/compiled.h

```diff
@@ -27,14 +27,14 @@ extern "C" {
 extern Obj InfoDecision;
 
 
-/* types, should go into 'gvars.c' and 'records.c' * * * * * * * * * * * * */
+// types, should go into 'gvars.c' and 'records.c' * * * * * * * * * * * *
 
 typedef UInt    GVar;
 
 typedef UInt    RNam;
 
 
-/* checks, should go into 'gap.c'  * * * * * * * * * * * * * * * * * * * * */
```

Comment:
If doing a cleanup, would it be possible to look for cases like this, where comments end in a line of stars, and clean those up?

Reply:
Found just a few, all in this file. Fixed now.